DOC-1661: added outbound links to RTC setup videos. (#2439)
tinymce/tinymce-docs: modules/ROOT/pages/rtc-getting-started.adoc

----

For details on the available configuration options, see: xref:rtc-options-overview.adoc[Configuration Options].

> An alternative to this documentation is a series of https://youtube.com/watch?v=zvu-7w-94vI&list=PLE6i0Bfo_RjlVyzACyh7XnnXJLoXD7h20[8 videos on setting up Real-Time Collaboration with TinyMCE]:

Reviewer: ```suggestion
An alternative to this documentation is a series of https://www.youtube.com/playlist?list=PLE6i0Bfo_RjlVyzACyh7XnnXJLoXD7h20[8 videos on setting up Real-Time Collaboration with TinyMCE]:
```
This might be a better link? As the current link points to the second video which is rather confusing.

Author: > As the current link points to the second video which is rather confusing.

I know I checked that at least once before committing. Obviously checking it twice was what I should have done. Thanks for catching that.

1. https://youtube.com/watch?v=x5wFUCs_-O8[Getting started with RTC configuration].
2. https://youtube.com/watch?v=zvu-7w-94vI[Configuring the client and server].
3. https://youtube.com/watch?v=pa_LPP21eHA[Adding collaborative editing to TinyMCE].

> 4. https://youtube.com/watch?v=gDOmhKpaUfQ[Removing locks on a CMS ].

Reviewer: ```suggestion
4. https://youtube.com/watch?v=gDOmhKpaUfQ[Removing locks on a CMS].
```

5. https://youtube.com/watch?v=s94DK72njik[Removing JS code not needed].
6. https://youtube.com/watch?v=2Teq6roNinY[Fortifying end-to-encryption].
7. https://youtube.com/watch?v=KMbj6MnJPWU[Restricting access in document editing].

> 8. https://youtube.com/watch?v=BULlF2aM5q0[Configuring additional settings for better real-time collaboration with TinyMCE ].

Reviewer: ```suggestion
8. https://youtube.com/watch?v=BULlF2aM5q0[Configuring additional settings for better real-time collaboration with TinyMCE].
```
